src/contributors/prompt.js

```diff
@@ -69,10 +69,28 @@ function getQuestions(options, username, contributions) {
   ]
 }
 
+function getValidUserContributions(options, contributions) {
+  const validContributionTypes = util.contributionTypes(options)
+  const userContributions = contributions && contributions.split(',')
+
+  const validUserContributions = _.filter(
+    userContribution => validContributionTypes[userContribution] !== undefined,
+  )(userContributions)
+
+  if (_.isEmpty(validUserContributions)) {
+    throw new Error('Invalid contribution type(s) entered')
```

Comment:
Could we make the error include a comma-separated list of the invalid contribution types?

Reply:
you mean to say, have a list of **valid** contribution types?

Reply:
I want them to be able to see which contribution type they entered which was invalid so they know not to enter that again.

Reply:
added please take a look :+1: